projects/components/src/lib/advanced-search/advanced-search.scss

```diff
@@ -3,17 +3,14 @@
 :host {
   display: flex;
   flex-direction: column;
+  overflow: auto;
 }
 
 .search-bar {
   display: flex;
   align-items: center;
   flex: 1;
-
-  button.new-filter {
-    height: 48px;
-    width: 48px;
-  }
+  padding: 10px 20px 10px 0;
```

Comment:
In general the app follows a gridline of 8px increments. In this case, the right padding should match the left padding of the container, which is `16px`, and the padding above can be `8px`

In order to match the height of added filters, we can just make it `height: 48px`.